### ISSUE TITLE
Research if microservices cover the entire functionality of duggaedservice.php

### ISSUE THREAD

**Issue by f22linhe** [HGustavs/LenaSYS#15434]
Review the microservices in the duggaedService folder. Do these microservices cover the entire functionality of the original service file, duggaedservice.php? If you find any functionalities that are not covered, do we need to create a new microservice for this functionality? Don´t forget to share your conclusion with the group.

https://github.com/HGustavs/LenaSYS/blob/g2-2024-v5/DuggaSys/duggaedservice.php

**Comment by c22mohel** [comment]
I have looked into duggaedService and I have concluded that these are the functionalities it performs. 

It can create dugga and duggaVariante, both handled by the microservices createDugga_ms.php and createDuggaVariant_ms.php. These manage the creation of new duggas, determined by the variable $opt where "SAVDUGGA" is for createDugga and $opt "ADDVARI" is for createDuggaVariant. 

duggaedService also handles updates for createDugga and duggaVariant. For dugga it has the same $opt and createDugga, this is done by checking the $qid variable; if it is empty, it inserts, but if not, it updates. In my opinion, this is a suboptimal solution; I believe these microservices need to be merged, or updateDugga needs its own $opt. Depending on the implementation of these microservices, this might not need to be changed, as $opt could become obsolete. However, at the moment, the functionality to updateDugga is covered by updateDugga_ms.php.


This is not the case for updating dugga variante. For updating dugga variante, it has its own $opt, which is "SAVVARI". This functionality is completely covered by updateDuggaVariant_ms.php.

duggaedService is also used to delete dugga and duggaVariante. This is handled by deleteDugga_ms.php and deleteDuggaVariant_ms.php respectively.

duggaedService is also used to retrieve information about a dugga. This is completely covered by retrieveDuggaService_ms.php. As far as I can tell, all the functionalities in duggaedService are covered by the microservices.
